### PR TITLE
Translate All November 2014 News (id)

### PR DESCRIPTION
**Pull request by meisyal**
I just finished one post. But, I am still translating the rest.
Please, review my work when it's ready @gozali.

Thank you so much.

### CODE REVIEW

File: id/news/_posts/2014-11-03-tropicalrb-2015-cfp.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+layout: news_post
+title: "Tropical Ruby 2015 CFP Dibuka"
+author: "Guilherme Cavalcanti"
+translator: "meisyal"
+date: 2014-11-03 15:20:57 +0000
+lang: en
```
Comment (marocchino) [inline id/news/_posts/2014-11-03-tropicalrb-2015-cfp.md]: en? 'ㅁ'

**Comment by meisyal** [comment]
@marocchino mistake. Thanks for reviewing.
I have pushed a new commit.

**Comment by gozali** [comment]
+1

thanks @meisyal

**Comment by meisyal** [comment]
I have done translating all November 2014 News into Bahasa Indonesia.
Please, review my work @gozali!
Thanks.

**Comment by stomar** [comment]
Please open a new PR:
* **never merge upstream/master into your topic branch**
* please squash the commits into a single commit or at least a history as clean as possible
* use a commit message conforming to our conventions, e.g. `Translate all November 2014 news (id)`

**Pull request closed without merging**